AIToolbox/experiment_save/result_package/abstract_result_packages.py

```diff
@@ -282,10 +282,10 @@ def create_other_object_pkg(other_object):
         """
 
         Args:
-            other_object (AIToolbox.experiment_save.abstract_result_package.AbstractResultPackage or dict):
+            other_object (AIToolbox.experiment_save.result_package.abstract_result_packages.AbstractResultPackage or dict):
 
         Returns:
-            AIToolbox.experiment_save.result_package.AbstractResultPackage:
+            AIToolbox.experiment_save.result_package.abstract_result_packages.AbstractResultPackage | AIToolbox.experiment_save.result_package.abstract_result_packages.MultipleResultPackageWrapper:
```

Comment:
Is this really the best way of specifying the OR for return types. MultipleResultPackageWrapper wouldn't be really needed as it is inherited from AbstractResultPackage but PyCharms type guessing fails if this is not there